server/server.js

```diff
@@ -0,0 +1,119 @@
+import express, { response } from 'express';     
+import cors from 'cors';
+const app = express();
+import { join, dirname } from 'path';
+import { Low, JSONFile } from 'lowdb';
+import { fileURLToPath } from 'url';
+import { v4 as uuidv4 } from 'uuid';
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+const file = join(__dirname, 'data/db.json');
+const adapter = new JSONFile(file);
+const db = new Low(adapter);
+
+await db.read();
+
+if (!db.data) db.data = { users: [] };
+
+app.use(express.json());
+app.use(cors());
+
+
+
+//* SIGN UP a NEW USER
+app.post("/signup", async (request, response)=> {
+console.log("new user signing up!");
```

Comment:
Here the code of the function should get a bit indented. Otherwise it is a bit hard to see, which code belongs to that route.